listen.go

```diff
@@ -50,6 +50,8 @@ func (l *RelayListener) Addr() net.Addr {
 }
 
 func (l *RelayListener) Multiaddr() ma.Multiaddr {
+	// TODO(lgierth): t0182 "default failure"
+	return ma.Cast(ma.CodeToVarint(P_CIRCUIT))
```

Comment:
wait, isn't there dead code below?

Reply:
Yeah I was trying to pin down the test failures -- this PR is by no means ready for merge yet